### ISSUE TITLE
[Bug] Could not connect to repository

### ISSUE THREAD

**Issue by matke-84** [elgatito/plugin.video.elementum#932]
When I install version 0.1.91 android arm, elementum installs normally, then it starts to install burst, after a few seconds it says that it couldn't install automatically. I go to elementum repository and it says could not connect to repository. I delete version 0.1.91, install version 0.1.87, everything works normally. As far as I know, this is not an isolated case, many people who do an installation from scratch complain.

**Comment by elgatito** [comment]
@matke-84 Any chance to to show a log file?

**Comment by matke-84** [comment]
I hope it's ok. If you need anything else...

https://paste.kodi.tv/zemonoleja

**Comment by elgatito** [comment]
@matke-84 

```
2023-07-07 17:23:01.744 T:7089  warning <general>: Repository add-on repository.elementum uses plain HTTP for add-on downloads in path http://:65220:65220/repository/elgatito/ - this is insecure and will make your Kodi installation vulnerable to attacks if enabled!
```

Yes, was just able to reproduce it on my side and fix. 
Thanks

**Comment by matke-84** [comment]
You're welcome. Thank you for continuing to maintain the best addon for movies and tv shows.

**Comment by elgatito** [comment]
Should be fixed in 0.1.92, but it can be that it would require removing repository.elementum from Kodi and then restarting Elementum/Kodi to install a proper repository.

**Comment by matke-84** [comment]
It works now. Thanks.
